### PR TITLE
Add two-batch / n-batch core nodes

### PR DESCRIPTION
**Pull request by sskutnik**
My first node commit! Note that LaTeX isn't rendering (yet).

### CODE REVIEW

File: reactivity-swing/info.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+uid: reactivity-swing
+title: Cycle reactivity swing
+prerequisites:
+  - define_reactivity
```
Comment (gonuke) [inline reactivity-swing/info.yml]: (I propose that) all PR's should only be considered valid if all prerequisites are satisfied, perhaps by "empty" boundary nodes.  If this is adopted, then this PR should also define `define_reactivity`.

**Comment by gonuke** [comment]
In my opinion, this node covers many different learning objectives.  Perhaps we can use it as an example of a node that gets subdivided in the future.

**Review by davis68 — COMMENTED** [review]
There's a few spots that have `uid` instead of `uuid`.  That's the older usage but we've standardized on `uuid` now.

**Comment by sskutnik** [comment]
@davis68 I've fixed up UUID; anything else you see that needs to be fixed up?

**Comment by sskutnik** [comment]
Handled via #65

**Pull request closed without merging**